File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Lab/04_Password_Guess.js
function passwordGuess(input) {
    let realPassword = "s3cr3t!P@ssw0rd"
    let correctPassMsg = "Welcome"
    let wrongPassMsg = "Wrong password!"

    let triedPassword = input[0];

    if (triedPassword == realPassword) {
        console.log(correctPassMsg);
    } else {
        console.log(wrongPassMsg);
    }
}

// passwordGuess(["s3cr3t!P@ssw0rd"]);
// passwordGuess(["qwerty"]);
// passwordGuess(["s3cr3t!P@sswd"]);

File: Python_Courses/Python_Basics/Python_PB_2024_06/02_Conditional_Statements_Lab/04_Password_Guess.py
# фразата "s3cr3t!P@ssw0rd".
# При съвпадение да се изведе "Welcome".
# При несъвпадение да се изведе "Wrong password!".

attempt = input()

if attempt == "s3cr3t!P@ssw0rd":
    print("Welcome")
else:
    print("Wrong password!")
